.github/CODEOWNERS

```diff
@@ -0,0 +1,33 @@
+# Core
+/airflow/executors/ @kaxil @XD-DENG @ashb @turbaszek
+/airflow/jobs/ @kaxil @ashb @XD-DENG
+/airflow/models/ @kaxil @XD-DENG @ashb @turbaszek
+
+# DAG Serialization
+/airflow/serialization/ @kaxil
+
+# Kubernetes
+/airflow/kubernetes/ @dimberman
+/airflow/kubernetes_executor_templates/ @dimberman
+/airflow/executors/celery_kubernetes_executor.py @dimberman
+/chart/ @dimberman
+
+# Docs
+/docs/ @mik-laj @kaxil @vikramkoka
+
+# API
+/airflow/api/ @mik-laj @kaxil
+/airflow/api_connexion/ @mik-laj @kaxil
+
+# WWW
+/airflow/www/ @ryanahamilton
+
+# Providers
+/airflow/providers/google/ @turbaszek @mik-laj
+
+# Dev tools
+/.github/workflows/ @potiuk @ashb @kaxil
+breeze @potiuk
+/dev/ @potiuk @ashb @kaxil
+/provider_packages/ @potiuk
+/scripts/ @potiuk
```

Comment:
Should we also add some of the providers - such as google/ amazon/ few others that have clear ownershiip? I am recently discussing it with AWS people (I sent them link to the PR) and we could also invite the Composer team for google provider and probably we can easily find other people for other parts

I guess those people do not have to be committers to be in CODEOWNERS? Or do they?

Reply:
And I also asked them to join the discussion and share their questions at the devlist :)

Reply:
> I guess those people do not have to be committers to be in CODEOWNERS? Or do they?

I'm afraid they have to be:
> The people you choose as code owners must have write permissions for the repository.

Reply:
It's a pity :(, This means that the AWS/Compose people should start working on their committer's status :D